pom.xml

```diff
@@ -105,6 +105,9 @@
         <module>example-runner</module>
         <module>performance</module>
         <module>deployment</module>
+        <!-- Commented out due to not being used
+        <module>deployment-jvm</module>
```

Comment:
why is it commented out? if we need it as a module, add the module

Reply:
This has been removed.